src/Storage/RevisionableStorageInterface.php

```diff
@@ -0,0 +1,40 @@
+<?php
+
+/**
+ * @file
+ * Contains \Drupal\entity\Storage\RevisionableStorageInterface.
+ */
+
+namespace Drupal\entity\Storage;
+
+use Drupal\Core\Entity\ContentEntityInterface;
+use Drupal\Core\Entity\EntityStorageInterface;
+
+/**
+ * Defines some additional storage methods used for revision support.
+ */
+interface RevisionableStorageInterface extends EntityStorageInterface {
```

Comment:
As this is for content entities, should this have Content in the name? Also, same as #5, put it in a Revision namespace?